### PR TITLE
Quick FR fixes

### PR DESCRIPTION
**Pull request by jesus2099**
### Summary

This is a…

- [ ] Correction
- [ ] Addition
- [ ] Restructuring
- [x] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

Some things appeared slightly unnatural in the French navigation (left sidebar and pagination buttons).

### Description of the Change

We would not select the French language (la langue française) but French as language (je choisis le français comme langue).
Also for consistency, I put infinitive verb.
*Précédent* (Previous) was OK but *Prochain* (Next) was quite strange.
So I replaced by the translation of previous page and next page.

### Additional Action Required

I was reviewing the bigger FR files when I stopped because I wondered if maybe those changes have to be done in something like Transifex, instead of a PR.
So I committed only the small files, in case it's not the way to go.
I am never keen to use Transifex, BTW.

### CODE REVIEW

File: _locale/fr/LC_MESSAGES/genindex.po [hunk 1]
```diff
@@ -21,4 +21,4 @@ msgstr ""
 
 #: ../../genindex.rst:4
 msgid "Index"
-msgstr "Indice"
+msgstr "Index"
```
Comment (jesus2099) [inline _locale/fr/LC_MESSAGES/genindex.po:24]: It's also Index in French.

File: _locale/fr/LC_MESSAGES/sphinx.po [hunk 2]
```diff
@@ -97,15 +97,15 @@ msgstr "Accueil du projet"
...
+msgstr "Les gabarits"
 
 #: ../../_templates/versions.html:42
 msgid "Options"
 msgstr "Options"
 
 #: ../../_templates/versions.html:48
 msgid "Select Version"
-msgstr "Sélectionnez la version"
+msgstr "Sélectionner la version"
```
Comment (jesus2099) [inline _locale/fr/LC_MESSAGES/sphinx.po:108]: For consistency with below's "**Choisir** la langue"

File: conf.py [hunk 3]
```diff
@@ -41,9 +41,9 @@
 default_language = 'en'
 supported_languages = [
     ('en', 'English'),
-    ('fr', 'Française'),
```
Comment (jesus2099) [inline conf.py:44]: It was really unusual, I think, to click a **Française** link.

**Comment by rdswift** [comment]
THANK-YOU!  I did all he translations for the whole document by hand (it's not on Transifex) using Google Translate because I don't speak French.  I'm not surprised in the least that there are errors, so I really appreciate any help in identifying them.

**Comment by jesus2099** [comment]
Great.
You're welcome.
So I will continue my review on big files, later. :)

**Comment by rdswift** [comment]
Thanks.  I am going to add your name to the contributors list in `about_picard/acknowledgements.html`, so would you prefer it to be displayed as "jesus2099" or something different?

**Comment by jesus2099** [comment]
Yes, just jesus2099, it's fine. 😉
I hope I can make the bigger review not so late...

**Pull request merged**